app/dashboard/templates/bounty/invoice.html

```diff
@@ -0,0 +1,231 @@
+{% load i18n static %}
+<!doctype html>
```

Comment:
Should we wrap any of the copy in this template in `trans` or do we only want English served here for now? @owocki

Reply:
just added trans tags